cartoframes/context.py

```diff
@@ -184,6 +190,53 @@ def _table_exists(self, table_name):
 
         return False
 
+    def _send_batches(self, df, table_name, temp_dir, geom_col, lnglat):
+        """Batch sending a dataframe"""
+        subtables = []
+        # send dataframe chunks to carto
+        for chunk_num, df_chunk in tqdm(df.groupby([i // MAX_IMPORT_ROWS
```

Comment:
Nope, every row needs to be categorized by this method. `0, 0, 0, 0, ..., 1, 1, 1, 1, ..., 2, 2,...`